components/Header/index.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+export default function Header() {
+  return (
+    <>
+      <StyledHeader>
+        <h1>Travel Journal</h1>
+      </StyledHeader>
+    </>
+  );
+}
+
+export const StyledHeader = styled.header`
+  padding: 12px;
+  overflow: hidden;
+  background-color: #f1f1f1;
+  position: relative;
+  text-align: center;
+  color: black;
+  text-decoration: none;
```

Comment:
As far as I can see, there is no text-decoration in your header that needs to be removed.